package.json

```diff
@@ -41,6 +42,24 @@
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-react": "^7.37.1",
     "globals": "^15.11.0",
-    "prettier": "3.3.3"
+    "husky": "^9.1.6",
+    "lint-staged": "^15.2.10",
+    "prettier": "3.3.3",
+    "prettierr": "^1.15.3-dev"
```

Comment:
what is this used for

Reply:
Husky allows you to add git hooks. Here, `pre-commit` is configured to run `lint-staged` every time I try to commit.
Lint-staged works together with Husky and runs linter and formatter only on changed files. Here, it runs `prettier` on all files with the `.jsx` extension.
Prettierr is a bug. Removed.
Done.